go/keybase/main.go

```diff
@@ -115,6 +115,9 @@ func mainInner(g *libkb.GlobalContext) error {
 	checkSystemUser(g.Log)
 
 	if !cl.IsService() {
+		if err = logger.SaveConsoleMode(); err == nil {
```

Comment:
This error will be overwritten below, is that intended?  It won't cause an exit out of this `mainInner`, which is unlike all of the other settings of `err`